### DISCUSSION TITLE
[Feature Request] Add an optional mouseover option that will display a DropDown

### DISCUSSION THREAD

**Discussion by timomedia** [heroui-inc/heroui#4694] [Feature requests]
### Is your feature request related to a problem? Please describe.

As the title says, I think you should add a hover option to display your Dropdown Components. This is quite common for using dropdowns to make menus along with Navbars

### Describe the solution you'd like

The title also describes the features of this section. Just hover the mouse over to display the dropdown and remove it to hide it like how multi-level menus work.

### Describe alternatives you've considered

You can see a few examples below.
https://ant.design/components/dropdown
https://daisyui.com/components/dropdown/

**Comment by J4v4Scr1pt** [comment]
+1

**Comment by blockfer-rp** [comment]
+1 Please much needed feature.

**Comment by shahzoab** [comment]
+1

**Comment by PaisWillie** [comment]
+1

**Comment by jimbycat** [comment]
+1
